packages/tizen_rpc_port/example/client/lib/message_client.dart

```diff
@@ -102,9 +102,8 @@ class Message extends ProxyBase {
     final Parcel parcel = Parcel.fromPort(port);
     final int cmd = parcel.readInt32();
     if (cmd != _MethodId.result.id) {
-      print('Received parcel is invalid. $cmd');
+      throw Exception('The received parcel is invalid ($cmd).');
     }
-
```

Comment:
@upple A code change has been made to auto-generated file. Please review and update the TIDL compiler.

Reply:
https://review.tizen.org/gerrit/c/platform/core/appfw/tidl/+/289709
I'm preparing the patch.

Reply:
Flutter exceptions are not like C# exceptions. When unhandled, they don't crash the process but are just logged to the console.

Reply:
Thank you.
As you said, it doesn't cause crash.

Reply:
@swift-kim 
Hello. I have a question.
When applying this patch, if a throw occurs in the `_coinsumeCommand()`, how does the subsequent operation proceed? We are proceeding with the while loop to receive invalid parcels. We are concerned that if a throw occurs, the subsequent operation will be damaged.

Reply:
Unhandled exceptions are propagated to callers, aborting execution of the rest of the code.

Reply:
This will cause a series of errors in the data you receive afterwards.
Through while loop, we are organizing the wrong parcel that occurred before.
This was our intention.

Reply:
It's an exceptional situation (invalid parcel), so an exception is thrown. I don't see any problem with this. Are you saying the invalid parcel is part of the normal execution flow? Then why did you add a `print` there and not just ignore the invalid (actually normal) parcel? Also how do you think of `Parcel.fromPort` and `parcel.readInt32`? They can both throw exceptions as well.

Reply:
A typical case in which the parcel is not valid is when the sender transmits late and the previous request is timeout.
In this case, an exception occurs in the fromPort and does not work afterwards, so when the parcel sent by the sender arrives later, it is received when the next request is processed. However, this should be ignored because it is a parcel for the previous request.
Printing log was simply put for debugging. If you think it's unnecessary, you can remove it.

Reply:
Okay, I'll remove the logic in the currently open PR (#543).

Reply:
Thank you.
Then, we will prepare above gerrit patch without these changes.

Reply:
No, you can't just remove the if statement from `_consumeCommand`. It will leave the `cmd` variable unused and the linter will complain again. You should check all the changes regarding the `tizen_rpc_port` package in my new PR and the TIDLC output should be exactly the same as the changed code. Note that EVERY change in the PR has a reason. If you don't understand why the change was made or if you disagree with the change, please leave a comment there.

Reply:
Ok. I'll apply these changes. 
Thanks for the preparing this patch!

Reply:
Thank you!